earthsim/__main__.py

```diff
@@ -0,0 +1,43 @@
+"""
+Prototype of a param script that can set global_params via the
+appropriate environment variable to execute an arbitrary command.
+
+Example usage:
+
+param -cmd 'jupyter nbconvert --execute GSSHA_Workflow_Batched_Example2.ipynb' -p rain_intensity=25 -p rain_duration=3600
```

Comment:
I wasn't sure what to call it: right now it is in ``earthsim`` but we hope to migrate it to ``param``. I'm happy to name it whatever you want though I don't know if an ``earthsim`` console script would have a long term future.

Reply:
Oh, I see -- I thought the command name would default to the module name, but I see later that it's explicitly declared ``param = earthsim.__main__:main``.  If it can really be supplied as an arbitrary command name, then `param` seems highly likely to conflict with something else on the user's path, so I think it should be something less generic.  Maybe `run_with_params`?  Not sure if running with params is more or less dangerous than running with scissors.

Reply:
I don't believe their is a unix/windows command called ``param`` so I don't see why param couldn't have a console script of that name. What do you think users might have on their path that conflicts with it? 

 If we have to worry about a clash, I suppose I might consider something like ``params`` or ``parameters`` instead.

Reply:
Both of those are equally generic. If the intent is for param to supply this command, then calling it `param` is reasonable.  @ceball, do you see any conflict with some other usage of a param module command? Seems ok to me.

Reply:
>  If the intent is for param to supply this command, then calling it ``param`` is reasonable ...

That was what I was thinking at least.